Userland/Libraries/LibGfx/GrayscaleBitmap.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2022, MacDue <macdue@dueutil.tech>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include "Size.h"
+#include <AK/RefCounted.h>
+#include <AK/RefPtr.h>
+#include <AK/StringView.h>
+
+namespace Gfx {
+
+class GrayscaleBitmap {
+public:
+    GrayscaleBitmap() = delete;
+    constexpr GrayscaleBitmap(ReadonlyBytes data, unsigned width, unsigned height)
+        : m_data(data)
+        , m_size(width, height)
+    {
```

Comment:
:thinking: Can we `VERIFY(width * height == data.size());`?

Reply:
Yep :+1: